src/main/java/io/anserini/search/SearchArgs.java

```diff
@@ -189,6 +189,34 @@
       usage = "RM3 parameter: print original and expanded queries")
   public boolean rm3_outputQuery = false;
 
+  // BM25 PRF Options
+  @Option(name = "-bm25prf", usage = "use bm25 PRF query expansion model")
```

Comment:
Can you document where you get these default parameters from? Just like how we document the source of the other parameter values?

Reply:
The k1 and b are copied from the bm25's default options. For others, we set them arbitrarily (actually suggested by Prof.Sakai, I will ask him if there is any ref about it).

Reply:
I talked to Prof.Sakai, and it seems that these values came from his experience on TREC collections, so there is no reference about it. These values are quite "conservative", so users are expected to perform some tuning to maximize its performance.

Reply:
Okay, great!